tests/foreman/ui/test_bookmark.py

```diff
@@ -635,7 +600,7 @@ def test_positive_delete_bookmark(self):
 
         :CaseImportance: Critical
         """
-        for entity in self.entities:
+        for entity in self.getOneEntity():
```

Comment:
That line was intended, we wanted to have 1 test which would try to create bookmarks for all possible entities and all the rest tests could use 1 random entity (to save some time as this scenario was already covered). Please revert it back.